mythril/laser/smt/bitvec.py

```diff
@@ -139,44 +139,48 @@ def __gt__(self, other) -> Bool:
         union = self.annotations + other.annotations
         return Bool(self.raw > other.raw, annotations=union)
 
-    def __eq__(self, other) -> Bool:
+    #MYPY: fix complains about overriding __eq__
```

Comment:
Usually there would be a space after # in comments from PEP8 standards.

Reply:
Fixed in next commit